### PR TITLE
fix: correct errors with accessibility navigation and fix small bugs

### PR DESCRIPTION
**Pull request by eemelioksanen**
## WHY:

Fixes bugs

## Check off each of the following tasks as they are completed

- [x] I have reviewed the changes myself in this PR. Please check the [self-review document](https://github.com/OpenVAA/voting-advice-application/blob/main/docs/contributing/self-review.md)
- [x] I have added or edited unit tests.
- [x] I have run the unit tests successfully.
- [x] I have run the e2e tests successfully.
- [x] I have tested this change on my own device.
- [x] I have tested this change on other devices (Using Browserstack is recommended).
- [x] I have tested my changes using the [WAVE extension](https://wave.webaim.org/extension/)
- [x] I have added documentation where necessary.
- [x] Is there an existing issue linked to this PR?

**Clean up your git commit history before submitting the pull request!**

### CODE REVIEW

File: frontend/src/lib/components/modal/TimedModal.svelte [hunk 1]
```diff
@@ -113,14 +113,20 @@ The dialog can be closed by pressing the `Escape` key.
...
-<dialog bind:this={modalContainer} class="modal dark:bg-white dark:bg-opacity-10" aria-modal>
-  <div class="modal-box">
-    <slot />
-    <progress
-      id="modal-progress"
-      class="w-56 progress progress-error absolute bottom-0 left-0 right-0"
-      value={$progressBarTimer}
-      max={timerDuration} />
-  </div>
+<!-- svelte-ignore a11y-no-noninteractive-tabindex -->
```
Comment (anolpe) [inline frontend/src/lib/components/modal/TimedModal.svelte:116]: Maybe there should be comment explaining why this is ignored?

File: frontend/src/lib/i18n/translations/en/candidateApp.json [hunk 2]
```diff
@@ -205,7 +205,9 @@
     "info": "You can supply your own translations for the languages the application is available in. If you don’t write your own translations, your original response will be shown."
   },
   "error": {
-    "userNoCandidateError": "Error: The logged in user has no candidate info tied to it."
+    "userNoCandidateError": "Error: The logged in user has no candidate info tied to it. Please contact the system admininstrator.",
```
Comment (anolpe) [inline frontend/src/lib/i18n/translations/en/candidateApp.json:208]: admininstrator -> administrator

File: frontend/src/lib/i18n/translations/en/candidateApp.json [hunk 3]
```diff
@@ -205,7 +205,9 @@
...
+    "candidateNoNominationError": "Error: The candidate associated with the logged in user has no nomination. Please contact the system admininstrator.",
```
Comment (anolpe) [inline frontend/src/lib/i18n/translations/en/candidateApp.json:209]: admininstrator -> administrator

File: frontend/src/lib/i18n/translations/en/candidateApp.json [hunk 4]
```diff
@@ -205,7 +205,9 @@
...
+    "nominationNoElectionError": "Error: The nominatinon associated with the candidate has no election. Please contact the system admininstrator."
```
Comment (anolpe) [inline frontend/src/lib/i18n/translations/en/candidateApp.json:210]: admininstrator -> administrator

**Review by anolpe — APPROVED** [review]

**Review by Henriki2305 — APPROVED** [review]

**Pull request merged**